src/nvim/menu.c

```diff
@@ -696,9 +700,11 @@ static dict_T *menu_get_recursive(const vimmenu_T *menu, int modes)
       if ((menu->modes & modes & (1 << bit)) != 0) {
         dict_T *impl = tv_dict_alloc();
         if (*menu->strings[bit] == NUL) {
-          tv_dict_add_str(impl, S_LEN("rhs"), (char *)"<Nop>");
+          tv_dict_add_str(impl, S_LEN("rhs"), (char *)"");
```

Comment:
No, just remove `if()`, what do you think `str2special_save` is going to do with the empty string and how exactly this is common to justify adding four more lines? `nvim_get_keymap` does not bother with treating empty string specially. Also the cast here is in any case useless.

Reply:
Eh right yeah, sorry for being dense :D I'll fix that.